src/main/app/controller/GetController.ts

```diff
@@ -44,7 +44,7 @@ export class GetController {
       ...content,
       sessionErrors,
       htmlLang: language,
-      isDraft: req.session?.userCase.state ? req.session.userCase.state === State.Draft : true,
+      isDraft: req.session?.userCase?.state ? req.session.userCase.state === State.Draft : true,
```

Comment:
Might be worth updating the type for this in `src/main/app/controller/AppRequest.ts` to be `userCase?: CaseWithId;`. This way TypeScript will highlight any other places were it might not be set.

Reply:
Was throwing a lot of errors mainly to do with the tests - will investigate separately so that I don't hold up merging this fix in.